File: dataset/output-bk/MISTRAL_codestral-latest/Task113_MISTRAL_codestral-latest.cpp
#include <iostream>
#include <string>

int main() {
    std::string input;
    std::string result = "";

    std::cout << "Enter strings to concatenate (enter 'end' to stop):
";
    std::getline(std::cin, input);

    while (input != "end") {
        result += input;
        std::getline(std::cin, input);
    }

    std::cout << "Concatenated string: " << result << std::endl;

    return 0;
}

File: dataset/output/MISTRAL_codestral-latest/Task113_MISTRAL_codestral-latest.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        StringBuilder sb = new StringBuilder();

        System.out.println("Enter strings to concatenate (enter 'end' to stop):");
        String input = scanner.nextLine();

        while (!input.equals("end")) {
            sb.append(input);
            input = scanner.nextLine();
        }

        System.out.println("Concatenated string: " + sb.toString());
    }
}